### PR TITLE
3.0 - Article 'How to troubleshoot the SSO'

### PR DESCRIPTION
**Pull request by bgouraud**

### CODE REVIEW

File: content/md/user-management/troubleshoot-the-sso.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+title: How to **troubleshoot** the Single Sign-On
+popular: false
+ee-only: true
+related: configure-saml-sso
+priority: low
+---
+
+# Overview
+
+Single Sign-On is a powerful tool for productivity but sometimes, as multiple applications are concerned, it can be a hassle to troubleshoot when something goes wrong with it.
```
Comment (LaureBro) [inline content/md/user-management/troubleshoot-the-sso.md]: you could simply say (at the end): "when something goes wrong." and remove "with it"

File: content/md/user-management/troubleshoot-the-sso.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Here are a few tips to help you understand what is going on between your `IdP` and your `SP`.
```
Comment (LaureBro) [inline content/md/user-management/troubleshoot-the-sso.md]: should we specify what IdP and SP stand for?

File: content/md/user-management/troubleshoot-the-sso.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+# Make sure the IdP configuration is OK
+
+First, make sure that the configuration you have in your Identity Provider is consistent with the information from your SP:
+* the `Entity ID` of the SP
+* the `ACS URL`
+
+Make sure that there is no typo or missing characters.
```
Comment (LaureBro) [inline content/md/user-management/troubleshoot-the-sso.md]: should we prioritise actions like 
1/ Make sure that there is no typo or missing characters.
2/ Ensure that the claims....
3/ Check if users are well created

Check this page for the format of lists: https://github.com/akeneo/pim-helpcenter/edit/master/content/md/productivity/projects-notifications.md

File: content/md/user-management/troubleshoot-the-sso.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+priority: low
+---
+
+# Overview
+
+Single Sign-On is a powerful tool for productivity but sometimes, as multiple applications are concerned, it can be a hassle to troubleshoot when something goes wrong with it.
+
+Here are a few tips to help you understand what is going on between your `IdP` and your `SP`.
+
+# Make sure the IdP configuration is OK
```
Comment (LaureBro) [inline content/md/user-management/troubleshoot-the-sso.md]: there is many "make sure" in this article, you could call your titles "Check your IdP configuration"

File: content/md/user-management/troubleshoot-the-sso.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+* the `Entity ID` of the SP
+* the `ACS URL`
+
+Make sure that there is no typo or missing characters.
+
+Ensure that the claims are defined in the IdP: at least an **akeneo_uid** attribute must be declared and must send the information that will be checked against the username in the PIM.
+
+Check if the users are created in your authentication server and are allowed to access the PIM.
+
+# Make sure the PIM configuration is OK
```
Comment (LaureBro) [inline content/md/user-management/troubleshoot-the-sso.md]: "Check your PIM configuration" ?

File: content/md/user-management/troubleshoot-the-sso.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+On the PIM's side, make sure that the information regarding the IdP are correct:
+* the `Entity ID` of the IdP
+* the `Sign-on URL`
+* the `Logout URL`
+* the certificate
+
+Check if the **akeneo_uid** information received from the `IdP` corresponds to real usernames in the PIM. If not, the SSO process will not be able to match the **akeneo_uid** with the username from the PIM and access will be denied.
+
+# How do I access my PIM?
```
Comment (LaureBro) [inline content/md/user-management/troubleshoot-the-sso.md]: "I can't access the PIM" ?

File: content/md/user-management/troubleshoot-the-sso.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+Enabling the Single Sign-On feature takes over the classic login process in the PIM. Each time a user will try to access the PIM, they will be redirected to the `IdP`.
+So, if the SSO server is not available (maintenance, network issues, etc.) or your PIM is not well configured, your users won't be able to log into the PIM, and you too, even if you are the administrator.
```
Comment (LaureBro) [inline content/md/user-management/troubleshoot-the-sso.md]: you could put some text in bold **SSO server is not available** or **your PIM is not well configured**your users "won't be able to log into the PIM, neither would you"

File: content/md/user-management/troubleshoot-the-sso.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+That is why we kept the classical login interface in the PIM for maintenance purposes. You can access it using the following URL: [https://your-pim-url/user/login](https://your-pim-url/user/login).
+
+:::info
+For now, this classical login route is not restricted to certain user so anyone can access it.
```
Comment (LaureBro) [inline content/md/user-management/troubleshoot-the-sso.md]: "users" instead of user?

File: content/md/user-management/troubleshoot-the-sso.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+:::
+
+# Have a look at the technical log files
+
+The Single Sign-On configuration page in the PIM allows you to download the technical log files regarding the SSO process authentication.
+The log files can be downloaded as a zip archive containing the logs for the previous days, one file per day, for the last 10 days.
```
Comment (LaureBro) [inline content/md/user-management/troubleshoot-the-sso.md]: Can you add where to download the logs ? (or a screenshot?)

File: content/md/user-management/troubleshoot-the-sso.md (hunk 2 above)
Comment (NolwennP) [reply content/md/user-management/troubleshoot-the-sso.md]: Maybe put a link to the definitions you did in the other article. ;)

File: content/md/user-management/troubleshoot-the-sso.md (hunk 1 above)
Comment (bgouraud) [reply content/md/user-management/troubleshoot-the-sso.md]: fixed

File: content/md/user-management/troubleshoot-the-sso.md (hunk 2 above)
Comment (bgouraud) [reply content/md/user-management/troubleshoot-the-sso.md]: Yep you're right. I will had a tip section about that.

File: content/md/user-management/troubleshoot-the-sso.md (hunk 4 above)
Comment (bgouraud) [reply content/md/user-management/troubleshoot-the-sso.md]: Yep. I changed the titles with "Check your IdP configuration" and "Make sur the PIM configuration is right"

File: content/md/user-management/troubleshoot-the-sso.md (hunk 8 above)
Comment (bgouraud) [reply content/md/user-management/troubleshoot-the-sso.md]: fixed

File: content/md/user-management/troubleshoot-the-sso.md (hunk 9 above)
Comment (bgouraud) [reply content/md/user-management/troubleshoot-the-sso.md]: Yes. A screenshot was added.

**Pull request merged**